pkg/dashboard/index.html

```diff
@@ -101,7 +101,7 @@ <h2 class="panel-title" translate>Servers</h2>
     <script id="dashboard-hosts-tmpl" type="x-template/mustache">
         {{#machines}}
             {{! state can be 'failed' or 'connected' }}
-            <a tabindex="0" data-address="{{address}}" class="list-group-item {{state}}"
+            <a role="link" tabindex="0" data-address="{{address}}" class="list-group-item {{state}}"
```

Comment:
This actually has an `href` on the next line, so role and tabindex aren't actually needed.

```suggestion
            <a data-address="{{address}}" class="list-group-item {{state}}"
```